src/lambda/expression.rs

```diff
@@ -22,6 +23,7 @@ pub enum Expression {
     Cache(Cache),
     Path(Box<Expression>, Vec<String>),
     Protect(Box<Expression>),
+    Js(Arc<Runtime>),
```

Comment:
- Move `Js` to `IO::JS`
- In TargetRuntime, hold an implementation of `WorkerIO`